server/web_templates.go

```diff
@@ -82,8 +83,9 @@ var indexTemplate = template.Must(template.New("index.html.tmpl").Parse(`
   <section>
     <p class="title-heading small"><strong>Locks</strong></p>
     {{ if .Locks }}
+    {{ $basePath := .CleanedBasePath }}
```

Comment:
I think you were missing this from your PR which caused the template to not render if there were any locks. You can't reference `.CleanedBasePath` from inside the `range` because `.` is redefined to be the element in the range. Thus you need to define a variable reference to it before entering the range.